packages/@aws-cdk/cloud-assembly-schema/lib/manifest.ts

```diff
@@ -30,7 +32,27 @@ export class Manifest {
   public static load(filePath: string): assembly.AssemblyManifest {
     const raw: assembly.AssemblyManifest = JSON.parse(fs.readFileSync(filePath, 'UTF-8'));
     Manifest.patchStackTags(raw);
-    Manifest.validate(raw);
+    Manifest._validate({ manifest: raw }, raw.version);
+    return raw;
+  }
+
+  /**
+   * Saves `assets.json` to a file.
+   * @param assetManifest - the manifest
+   * @param filePath - the output file
+   */
+  public static saveAssetManifest(assetManifest: assets.ManifestFile, filePath: string) {
```

Comment:
I don't like the asymmetry in `save` and `saveAssetManifest`...since this is a stable module, we are going to be stuck with it :\
 
Can we make it cleaner?, like having a dedicated file called `assets.ts` that has a `save` and a `load` function. And maybe an `assembly.ts` file that exposes the top level version (`Assembly.version()`).

Reply:
Deprecated the original methods for `save/loadAssemblyManifest`